### ISSUE TITLE
multiple lists or tags: but entry is shown only once

### ISSUE THREAD

**Issue by aisbergde** [mpcjanssen/simpletask-android#1072]
I can assign multiple tasks (or lists) to an entry.
But each entry is shown only once.

I would expect an entry to appear multiple times under different tags or lists: for each assigned list or tag. This is a typical many-to-many-relationship

**Comment by aisbergde** [comment]
or an option where I can control this (maybe other users are minded differently)

**Comment by smichel17** [comment]
Duplicate of #153
